tests/integration/integration.test.ts

```diff
@@ -51,7 +51,7 @@ it('accepts a port to run on', () => {
 
   return new Promise<void>((done) => {
     exec(
-      'cd demo && npm run storybook:axeOnly -- --port 8111',
+      'cd demo && npm run storybook:axeOnly -- --port 8112',
```

Comment:
Changed the port because it turns out Chrome is already using 8111 (at least on my machine).